### ISSUE TITLE
QG checks (Release 24.03)

### ISSUE THREAD

**Issue by FaGru3n** [eclipse-tractusx/bpdm#757]
## QG checks
Please keep this issue open until QG is concluded and will be managed by the Issue Creator!
We will inform you about finding and proposals in separated issues, this issue here is for the Overview of the Checks!

### Please keep this issue open until QG is concluded!

Product Name: BPDM
Product Owner: @rybtim 
Dev SPOC: @nicoprow 
Helm Chart Version: ~4.0.0~ -> [4.0.2](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-4.0.2)
App Version: [5.0.0](https://github.com/eclipse-tractusx/bpdm/blob/bpdm-4.0.2/charts/bpdm/Chart.yaml#L26)
Branch: https://github.com/eclipse-tractusx/bpdm/tree/release/5.0.x

QG Approval: yes

Release Managemnet Reference Issue:   

### Check of Tractus-X Release Guidelines

- Currently implemented automatic checks can be found under your product on our [Release Guidelines Checks Board](https://eclipse-tractusx.github.io/sig-release/) 
- This QG Check is depending on the mandatory information from our current [Release Guidelines](https://eclipse-tractusx.github.io/docs/release)

#### TRG 1 Documentation

- [x] [TRG 1.01](https://eclipse-tractusx.github.io/docs/release/trg-1/trg-1-1) appropriate `README.md` 
- [x] #759
- [x] #762
- [x] #760

#### TRG 2 Git

- [x] [TRG 2.01](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-1) default branch is named `main`
- [x] [TRG 2.03](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-3) repository structure
- [x] [TRG 2.04](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-4) leading product repository
- [x] [TRG 2.05](https://eclipse-tractusx.github.io/docs/release/trg-2/trg-2-5) `.tractusx` metafile in a proper format

#### TRG 3 Kubernetes

- [x] [TRG 3.02](https://eclipse-tractusx.github.io/docs/release/trg-3/trg-3-2) persistent volume and persistent volume claim is used when needed

#### TRG 4 Container

- [x] [TRG 4.01](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-01) [semantic versioning](https://semver.org/) and tagging 
- [x] [TRG 4.02](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-02) base image is agreed  
- [x] [TRG 4.03](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-03) image has `USER` command and Non Root Container
- [x] [TRG 4.05](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-05) released image must be placed in `DockerHub`, remove `GHCR` references 
- [x] [TRG 4.06](https://eclipse-tractusx.github.io/docs/release/trg-4/trg-4-06) separate notice file for `DockerHub` has all necessary information

#### TRG 5 Helm

- [x] #786
- [x] [TRG 5.02](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-02) Helm chart location in `/charts` directory and correct structure
- [x] [TRG 5.03](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-03) proper version strategy 
- [x] #767
- [x] [TRG 5.06](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-06) Application must be configurable through the Helm chart 
- [x] #764 will be manged after current release cycle
- [x] [TRG 5.08](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-08) Product has a single deployable helm chart that contains all components 
- [x] #765
- [x] [TRG 5.10](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-10) Products need to support 3 versions at a time
- [x] [TRG 5.11](https://eclipse-tractusx.github.io/docs/release/trg-5/trg-5-11) Upgradeability
 
#### TRG 6 Released Helm Chart

- [x] [TRG 6.01](https://eclipse-tractusx.github.io/docs/release/trg-6/trg-6-1) Released Helm Chart 

#### TRG 7 Open Source Governance
- [x] [TRG 7.01](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-01) Legal Documentation
- [x] #761
- [x] [TRG 7.03](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-03) IP checks for project content 
- [x] #768
- [x] [TRG 7.05](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-05) Legal information for distributions
- [x] [TRG 7.06](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-06) Legal information for end user content
- [x] [TRG 7.07](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-07) Legal notice for documentation
- [x] [TRG 7.08](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-08) Legal notice for KIT documentation

**Comment by nicoprow** [comment]
Helm Chart Version: 4.0.0
App Version: 5.0.0

**Comment by FaGru3n** [comment]
Hi @nicoprow finished the first check on QG process and pointed every found thing into separate issues, main parts is licence directoty, some updated dependencies for postgres and some diagrams i found that are not editable in the way we described it in the TRG.

Some quickfixes i alreadey created as a little pr so thanks for cooperation and please recheck the found issues. 

Thanks in advance.

**Comment by nicoprow** [comment]
> Hi @nicoprow thanks for you workd so far.
> 
> should we also update the Chart version for the current release or will you add the license information and make another release?
> 
> Helm Chart Version: 4.0.0 -> [4.0.1](https://github.com/eclipse-tractusx/bpdm/releases/tag/bpdm-4.0.1) App Version: [5.0.0](https://github.com/eclipse-tractusx/bpdm/blob/bpdm-4.0.1/charts/bpdm/Chart.yaml#L26)

Yes please, the update contains the update for the quality gate checks

**Comment by RolaH1t** [comment]
@nicoprow with @FaGru3n on vacation this week => can you pls work with a proxy in his team to finally conclude this TRG issue.
It is the only thing preventing a QG approval for BPDM Golden Record.
Reach out to @Siegfriedk in case of contact questions. 
Thx, Roland

**Comment by nicoprow** [comment]
@RolaH1t All the points have been marked and I'm not aware of any blocking issue at the moment. What is still to do?

**Comment by RolaH1t** [comment]
@Siegfriedk can you pls confirm with @nicoprow that this issue can fully be closed?!
I only see TRG 5.07 (wrt dependencies): the box has been checked, but [#764](https://github.com/eclipse-tractusx/bpdm/issues/764) is not closed.
Once this is concluded, please check the final box in [#496](https://github.com/eclipse-tractusx/sig-release/issues/496)

**Comment by tomaszbarwicki** [comment]
Based on following feedback https://github.com/eclipse-tractusx/bpdm/issues/764#issuecomment-1980445083 reopening the QG checks.

**Comment by FaGru3n** [comment]
Hi everyone, 

after the current release 24.03 will close this qg-check issues but the version for postgresql is still open for the next release 24.05
so please be aware of this for the next release 
https://github.com/eclipse-tractusx/bpdm/issues/764

thanks for your help and your work
